frontend/src/Components/UserLogin/FormField/FormField.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import FormFieldI from '../../../Interfaces/UserSignUp/FormField';
+import './FormField.css'
+
+class FormField extends React.Component<FormFieldI,{ value: string },{}> {
+    constructor(props: FormFieldI) {
+        super(props);
+        this.state = {value: ''};
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(event: React.ChangeEvent<any>) {
+        this.setState({value: event.target.value});
+    }
+
+    handleSubmit(event: React.ChangeEvent<any>) {
+        //alert('A name was submitted: ' + this.state.value);
+        event.preventDefault();
+    }
```

Comment:
handleSubmit -> move this function to UserLogin having access to every field value.